src/ucp/wireup/wireup.c

```diff
@@ -1343,25 +1343,166 @@ static void ucp_wireup_discard_uct_eps(ucp_ep_h ep, uct_ep_h *uct_eps,
     }
 }
 
+static int
+ucp_wireup_are_all_lanes_p2p(ucp_ep_h ep, const ucp_ep_config_key_t *key)
```

Comment:
On the one hand, this func is only used in `wireup`.
On the other hand, there's nothing "wireup-specific" here, it can also reside in `ucp_ep.c` and be used from other places, if needed, and it is anywhere not inlined. wdyt?

Reply:
bare in mind that it's a temp function which will be removed next PRs, so not sure it's relevant